client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkUtils.java

```diff
@@ -606,4 +608,22 @@ public static void invalidateSerializedGetReducerFileGroupResponse(Integer shuff
           return null;
         });
   }
+
+  public static void addWriterShuffleIdsToBeCleaned(
+      LifecycleManager lifecycleManager, String appShuffleIdentifier) {
+    FailedShuffleCleaner.setLifecycleManager(lifecycleManager);
+    FailedShuffleCleaner.addShuffleIdToBeCleaned(appShuffleIdentifier);
+  }
+
+  public static void addShuffleIdRefCount(
+      LifecycleManager lifecycleManager, int celebornShuffeId, String appShuffleIdentifier) {
```

Comment:
Here is a typo. celebornShuffeId --> celebornShuffleId

Reply:
oops, fixed